### DISCUSSION TITLE
Problem with raman.x

### DISCUSSION THREAD

**Discussion by eamouj78** [nguyen-group/QERaman#3] [Q&A] [answered]
Dear developers,

Firstly, thank you for this nice code. I am trying to reproduce the results of graphene. Everything goes well until the last step, where I execute the 'raman.x' executable. I get an error:

forrtl: severe (24): end-of-file during read, unit 101, file /home/users/eamouj78/graphene/RAMAN/graphene.dvec
Image              PC                Routine            Line        Source
raman.x            0000000000420D2B  Unknown               Unknown  Unknown
raman.x            0000000000439C98  Unknown               Unknown  Unknown
raman.x            0000000000411942  raman_read_mp_rea         166  raman_read.f90
raman.x            000000000040481D  MAIN__                     51  raman.f90
raman.x            00000000004040E2  Unknown               Unknown  Unknown
libc-2.19.so       00007FFFEB598B05  __libc_start_main     Unknown  Unknown
raman.x            0000000000403FE9  Unknown               Unknown  Unknown

I am not sure what is happening here. The file 'graphene.dvec' was correctly reproduced. Any help would be appreciated.
Cany anyone helo me in this please

**Comment by nguyen-group** [comment]
Hi, if you can open 'graphene.dvec', and you can see the data like:
<img width="1344" alt="Screenshot 2023-10-20 at 1 29 12" src="https://github.com/nguyen-group/QERaman/assets/46996256/5e046391-ba10-4081-84f8-85f096c9874e">
Then file graphene.dvec should be correct. The problem might come from the compiler raman.x
Which compiler that you used to make quantum espresso?

**Reply by eamouj78** [reply, marked as answer]
Thanks very much for your response.

I have realized that there was a problem reading the wavefunctions in the pw.x step, which affected the Raman spectra file at the end.
I have managed to solve the problem.

**Reply by nguyen-group** [reply]
Good to hear that you solved the problem.
